src/ResourceManager/Compute/ChangeLog.md

```diff
@@ -19,6 +19,8 @@
 -->
 ## Current Release
 
+* Fix the bug of Get cmdlets, where the output did not include all of the results from next page links, if the result contains more than 120 items.
```

Comment:
How about:  Fixed bug to allow Get- cmdlets to return more than 120 results?  Does this apply to all compute get cmdlets, or just for container services or vmss?